index.js

```diff
@@ -18,13 +20,13 @@
 
 function paginate(query, options, callback) {
   query = query || {};
-  options = Object.assign({}, paginate.options, options);
-  let select = options.select;
+  options = options || {};
+  let select = options.select || null;
   let sort = options.sort;
   let populate = options.populate;
   let lean = options.lean || false;
-  let leanWithId = options.leanWithId ? options.leanWithId : true;
-  let limit = options.limit ? options.limit : 10;
+  let leanWithId = options.leanWithId || false;
```

Comment:
Most people don't need an "id" specifically, they just need { "lean" : true }, and to access _id, whether it is id or _id. Thoughts?